tests/TestCase/I18n/TranslatorRegistryTest.php

```diff
@@ -56,7 +57,7 @@ public function testGetNullPackageInitializationFromCache(): void
             ->willReturn($package);
 
         $cacheEngineNullPackage
-            ->method('read')
+            ->method('get')
```

Comment:
No idea why `read` was mocked on that class since it doesn't exist neither is it called.